lib/api_auth/request_drivers/rest_client.rb

```diff
@@ -12,6 +12,7 @@ class RestClientRequest # :nodoc:
       def initialize(request)
         @request = request
         @headers = fetch_headers
+        @processed_headers = capitalize_keys @request.processed_headers
```

Comment:
It looks like the moved to using `processed_headers` now as well here: https://github.com/mgomes/api_auth/commit/8de9ec93932a0ebd33aa456fa265a6489e4c9a16

And in master: https://github.com/mgomes/api_auth/blob/master/lib/api_auth/request_drivers/rest_client.rb#L49-L51